src/libraries/System.Security.Cryptography.X509Certificates/tests/PemTestData.cs

```diff
@@ -0,0 +1,275 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    internal static class PemTestData
```

Comment:
Why a new class? Can't these just go in TestData?

Reply:
No strong reason. Will consolidate.